### DISCUSSION TITLE
Setting up the story

### DISCUSSION THREAD

**Discussion by ajleach9689** [jeffaroundtheworld/2024gamecollab#71] [General]
Just wondering how we are going to set up the back story of the game? Like if were just going to do a click-through text scene or something more complicated?

**Comment by kodriq** [comment]
I honestly figured we wouldn't be able to explain much of it at the moment, but one thing i did think of could be files/papers lying around that you can interact with and read, could explain the fact they are being experimented on and that the monsters are students or smth
